File: NybSys.Models/ViewModels/VMAdvancedReport.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace NybSys.Models.ViewModels
{
  public  class VMAdvancedReport
    {
        public int EmployeeID { get; set; }
        public int DriverID { get; set; }
        public int VehicleID { get; set; }

        public string EmployeeName { get; set; }
        public string DriverName { get; set; }
        public string VehicleName { get; set; }
        public DateTime FromDate { get; set; }
        public DateTime ToDate { get; set; }
        public int PageIndex { get; set; }
        public int PageSize { get; set; }
    }
}

File: Client/src/app/Models/VMAdvancedReport.ts
import { Pagination } from "./Pagination";
 export class VMAdvancedReport implements Pagination{
     constructor()
     {
       this.EmployeeID=0;
       this.DriverID=0;
       this.VehicleID=0;
       this.FromDate=new  Date().toISOString().slice(0,10);
       this.ToDate=new  Date().toISOString().slice(0,10);

       
     }
     EmployeeID:number;
     DriverID:number;
     VehicleID:number;
     FromDate:string;
     ToDate:string;
     PageIndex: number;
    PageSize: number;


 }